po/R-fr.po

```diff
@@ -0,0 +1,645 @@
+"formats."
+
+#: date_standardization_helpers.R:408
+msgid ""
+"Please specify the formats encountered in your column of interest via the {."
+"emph format} argument."
+msgstr ""
+"Veuillez fournir les formats retrouvés dans la colonne d'intéret via le "
+"paramètre {.emph format}."
+
+#: date_standardization_helpers.R:549
+msgid "Unexpected data type provided to {.fn date_guess} function."
+msgstr ""
+"Le type de donnée fourni à la fonction {.fn date_guess} est différent du "
+"type que la fonction devait prendre."
+
+#: date_standardization_helpers.R:550
+msgid ""
+"You can convert the values into {.cls character} to enable format guessing."
+msgstr ""
+"Vous pouvez convertir les valeures en {.cls charactère} pour détecter leur "
+"format."
+
+#: date_standardization_helpers.R:551
+msgid ""
+"You've tried to guess the date format from values of type other than Date "
+"and character."
+msgstr ""
+"Vous avez tenté de détecter le format de valeures qui ne sont ni de type "
+"Date ou charactère."
+
+#: date_standardization_helpers.R:570
+msgid "Unable to match formats to target columns."
+msgstr ""
+"Impossible d'établir la correspondence entre formats et colonnes ciblées."
+
+#: date_standardization_helpers.R:571
+msgid ""
+"The number of target columns does not match the number of specified formats."
+msgstr ""
+"Le nombre de colonnes ciblées ne correspond pas au nombre de formats qui "
+"sont forunis"
+
+#: date_standardization_helpers.R:572
+msgid ""
+"Only one format is needed if all target columns contain values of the same "
+"format. Otherwise, one format per target column must be provided."
+msgstr ""
+"Seul un format est requis si les colonnes ciblées ont des valeures du même "
+"format. Sinon, vous devez fornir un format par colonne cible."
+
+#: date_standardization_helpers.R:577
+msgid "Target columns will be standardized using the format: {.val {format}}."
+msgstr ""
+"Les colonnes ciblées seront standardisées en utilisant le format suivant: {."
+"val {format}}."
+
+#: default_cleanepi_settings.R:48
+msgid ""
+"Found the following unrecognised arguments to {.fn clean_data}: {.emph "
+"{toString(extra)}}."
+msgstr ""
+"{.fn clean_data} ne reconnait pas les arguments suivants: {.emph "
+"{toString(extra)}}."
+
+#: default_cleanepi_settings.R:49
+msgid "{.fn clean_data} does not support arguments other than the defaults."
+msgstr ""
+"{.fn clean_data} ne prend pas d'arguments autre que ceux qui sont définis "
+"par défault."
+
+#: default_cleanepi_settings.R:50
+msgid "You provided unexpected arguments."
+msgstr "Vous avez fourni des arguments qui ne sont pas prévus."
+
+#: default_cleanepi_settings.R:51
+msgid ""
+"Run {.code get_default_params()} to display the list of default parameters."
+msgstr ""
+"Entrez {.code get_default_params()} pour consulter la list des arguments "
+"prédéfinis."
+
+#: dictionary_based_cleaning.R:48
+msgid "Incorrect data dictionary."
+msgstr "Dictionnaire de données incorrect."
```

Comment:
Not a translation comment but what does "incorrect" mean here? Could the message state more precisely what the issue is (e.g., invalid format, etc.)?

Reply:
I have updated the messages